src/quote/Quote.ts

```diff
@@ -40,6 +50,14 @@ export function validateQuoteSchema(
   return !!result
 }
 
+/**
+ * Deserializes a [[Quote]] object into a readable format.
```

Comment:
Change request:
If this function deserializes as mentioned in line 54, then the input is not already deserialized. 
Also, it's mentioned that `deserializedQuote` is a `serialized` object, isn't that contradictory?

Reply:
I have updated it. 

But, I am struggling with what I should write.

Reply:
Maybe some confusion on serialization/deserialiation?

Serialization is this process:
JS object ---> JSON object (string) // via `JSON.stringify`

Deserialization is this:
JSON object (string) --> JS object // via `JSON.parse`

What your function does is just transformation; it's designed to be called with an argument that has already been deserialized. You know this because you know how the function will be used; but the function itself doesn't need to mention that. Except in the code example maybe, where this use case can be illustrated.

--> Check out the code example and the doc of `fromAttestedClaim`, it will help.

Reply:
What do you think of this iteration?